test/zeta-aca-test_controllerScript/run.py

```diff
@@ -0,0 +1,132 @@
+            t.banner_timeout = timeout
+            t.connect(username=user, password=password)
+            sftp = paramiko.SFTPClient.from_transport(t)
+            sftp.put(local_path, server_path)
+            t.close()
+        return True
+    except Exception as e:
+        print(e)
+        return False
+
+
+# Execute remote SSH commands
+def exec_sshCommand_aca(host, user, password, cmd, timeout=10):
+    """
+    :param host
+    :param user
+    :param password
+    :param cmd
+    :param seconds
+    :return: dict
+    """
+    result = {'status': [], 'data': []}  # Record return result
+    try:
+        # Create a SSHClient instance
+        ssh = paramiko.SSHClient()
+        ssh.banner_timeout = timeout
+        # set host key
+        ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+        # Connect to remote server
+        ssh.connect(host, 22, user, password, timeout=timeout)
+        for command in cmd:
+            # execute command
+            stdin, stdout, stderr = ssh.exec_command(command, get_pty=True, timeout=timeout)
+            # If need password
+            if 'sudo' in command:
+                stdin.write(password + '\n')
+            # result of execution,return a list
+            # out1 = stdout.readlines()
+            out2 = stdout.read()
+            # execution state:0 means success,1 means failure
+            channel = stdout.channel
+            status = channel.recv_exit_status()
+            result['status'].append(status)
+            # result['data'].append(out1)
+            print(out2.decode())
+        ssh.close()  # close ssh connection
+        return result
+    except Exception as e:
+        print(e)
+        return False
+
+
+def talk_to_zeta(file_path, zgc_nodes_url):
+    with open(file_path, 'r', encoding='utf8')as fp:
+        zeta_data = json.load(fp)
+
+    # create ZGC
+    ZGC_data = json.dumps(zeta_data["ZGC_data"])
+    requests.post(zgc_nodes_url[0] + "/zgcs", ZGC_data)
+    requests.post(zgc_nodes_url[1] + "/zgcs", ZGC_data)
+
+    # add VPC
+    for tem in zeta_data["VPC_data"]:
+        VPC_data = json.dumps(tem)
+        vpc_response_data1 = requests.post(zgc_nodes_url[0] + "/vpcs", VPC_data).json()
+        vpc_response_data2 = requests.post(zgc_nodes_url[1] + "/vpcs", VPC_data).json()
+        video["zgc_id"] = vpc_response_data1["zgc_id"]
+        print(vpc_response_data1["zgc_id"])
+
+    # notify ZGC the ports created on each ACA
+    PORT_data = json.dumps(zeta_data["PORT_data"])
+    requests.post(zgc_nodes_url[0] + "/ports", PORT_data)
+    requests.post(zgc_nodes_url[1] + "/ports", PORT_data)
+    # TODO: 分别生成CHILD和PARENT的配置文件
+
+
+def run():
+    # # Call zeta API to create ZGC,vpc etc.and generate the information ACA need, and save it in zetaToAca_data.json
+    file_path = './data/zeta_data.json'
+    zgc1_nodes_url = ["http://172.16.62.247:8080", "http://172.16.62.248:8080"]
+    talk_to_zeta(file_path, zgc1_nodes_url)
```

Comment:
This is actually not what you should do with the zgc_node_url(ip), like calling /zgc, /vpc in the `talk_to_zeta`, you also want to call the /nodes POST api for these zgc_node_url. 

You should add a few entries of data in your `zeta_data.json`:
- zeta_api_ip (where you will actually call the /zgc, /vpc, /nodes APIs)
- NODE_data, with the format that the /nodes POST api desires [(refer to here)](https://github.com/futurewei-cloud/zeta/blob/main/docs/design/zeta_system_design.md#7331-add-zgc-node)

Then, in the `talk_to_zeta`, after creating the calling /zgc and /vpc, call /nodes POST to add the zgc1_nodes_url, can you refer to [here](https://github.com/futurewei-cloud/zeta/blob/main/deploy/install/deploy_wellcheck.sh#L102) for how to make the call.